cypress/fixtures/products.json

```diff
@@ -2,23 +2,32 @@
   "products": [
       {
           "id": 1,
-          "category": "Starter",
+          "category": "Starters",
           "name": "Pizza",
           "description": "This is a cheesy pizza",
           "price": 70
       },
       {
           "id": 2,
-          
+          "category": "Main Courses",
```

Comment:
are u sure this is how it will be sent off from the backend?

Reply:
category: [:starters, :maincourses, :desserts, :drinks